concordium-std/src/lib.rs

```diff
@@ -41,8 +30,9 @@
 //! This library has the following features:
 //! [`std`](#std-build-with-the-rust-standard-library),
 //! [`build-schema`](#build-schema-build-for-generating-a-module-schema),
-//! [`wasm-test`](#wasm-test-build-for-testing-in-wasm), and
-//! [`crypto-primitives`][crypto-feature].
+//! [`wasm-test`](#wasm-test-build-for-testing-in-wasm),
+//! [`crypto-primitives`][crypto-feature], and
+//! [`use-wee-alloc`](#use-a-custom-allocator)
```

Comment:
I would prefer if we followed the naming convention set by the Rust library team and call it `wee-alloc`.
They explicitly mention not to use `use-` as a prefix for an optional dependency https://rust-lang.github.io/api-guidelines/naming.html#feature-names-are-free-of-placeholder-words-c-feature

Reply:
Well "Do not include words in the name of a [Cargo feature](http://doc.crates.io/manifest.html#the-features-section) that convey zero meaning". 

I don't think `use` conveys zero meaning :P

Reply:
Admittedly `set` would be even more apt.

Reply:
The feature is not for using the optional dependency, it is for setting a global allocator.